src/components/interface/MellowLPTable/components/MellowLPEntry.tsx

```diff
@@ -0,0 +1,92 @@
+import { Agents } from "@contexts";
+import { Typography } from "@mui/material";
+import { Box, SystemStyleObject, Theme } from "@mui/system";
+import { AugmentedMellowLpVault } from "@utilities";
+import PoolField from "../../../composite/PoolField/PoolField";
+import LPMellowVaultInfo from "./LPMellowVaultInfo";
+import { ReactComponent as Mellow } from './mellow-icon.svg';
+import MellowLPPosition from "./MellowLPPosition";
+
+
+export type MellowLPEntryProps = {
+    onSelectItem: () => void;
+    lpVault: AugmentedMellowLpVault;
+    disabled: boolean;
+}
+
+const MellowLPEntry: React.FunctionComponent<MellowLPEntryProps> = ({ lpVault, onSelectItem, disabled }: MellowLPEntryProps) => {
+    const boxStyles: SystemStyleObject<Theme> = {
+        display: "flex",
+        borderRadius: "4px",
+        padding: "4px 8px",
+        marginTop: "16px",
+        background: "#472043",
+        marginLeft: "8px",
+        width: "100px",
+        justifyContent: "center",
+    }
+
+    const tagStyles: SystemStyleObject<Theme> = {
+        fontSize: '14px',
+        color:"#FF4AA9"
+    };
+
+    const titleStyles: SystemStyleObject<Theme> = { 
+        fontSize: '24px', 
+        lineHeight: '1.3', 
+        color: '#E5E1F9',
+        fontWeight: '700',
+        marginLeft: "16px",
+    };
+
+    const copyStyles: SystemStyleObject<Theme> = {
+        fontSize: '14px',
+        color: '#9B97AD', 
+
+        marginLeft: "8px",
+        marginTop: "16px",
+    }
```

Comment:
please move the styles into ./styles.ts file nested in same directory as this component, let's keep the component as clean as possible

Reply:
Resolved. https://github.com/Voltz-Protocol/voltz-ui/commit/a44c3c48e2a2660e1fc14f6eeeda646152d2edea